dashboard/app/controllers/admin_reports_controller.rb

```diff
@@ -270,26 +270,27 @@ def hoc_signups
   def retention
     require 'cdo/properties'
     SeamlessDatabasePool.use_persistent_read_connection do
-      @scripts = params[:scripts_ids].present? ?
-        params[:scripts_ids].split(',').map(&:to_i) :
-        [1, 17, 18, 19, 23]  # Default to the CSF scripts.
-      # Get the cached retention_stats from the DB, trimming those stats to only the requested
+      @all_scripts_names_ids = Script.order(:id).pluck(:name, :id).to_h
```

Comment:
Can we not do something like `Scripts.where(:name in selected_scripts_names)` rather than grabbing every script?